### PR TITLE
test: add coveralls integration

### PR DESCRIPTION
**Pull request by amagyar-iohk**
# Overview

Enables coveralls integration

Fixes #<issue number>

### My PR contains...
* [ ] No code changes (changes to documentation, CI, metadata, etc.)
* [ ] Bug fixes (non-breaking change which fixes an issue)
* [ ] Improvements (misc. changes to existing features)
* [ ] Features (non-breaking change which adds functionality)

### My changes...
* [ ] are breaking changes
* [ ] are not breaking changes
* [ ] If yes to the above: I have updated the documentation accordingly

### Documentation
* [ ] My changes do not require a change to the project documentation
* [ ] My changes require a change to the project documentation
* [ ] If yes to the above: I have updated the documentation accordingly

### Tests
* [ ] My changes can not or do not need to be tested
* [ ] My changes can and should be tested by unit and/or integration tests
* [ ] If yes to the above: I have added tests to cover my changes
* [ ] If yes to the above: I have taken care to cover edge cases in my tests

### CODE REVIEW

File: build_test.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+rm -rf "$LCOV_DIR"
+mkdir "$LCOV_DIR"
+
+# Run build and test
+echo "Running build and test"
+xcodebuild -scheme "AtalaPRISMSDK-Package" \
+    -destination "$DESTINATION" \
+    -derivedDataPath "$DERIVED_DATA_DIR" \
+    -enableCodeCoverage YES \
+    -quiet clean build test
```
Comment (goncalo-frade-iohk) [inline build_test.sh:29]: Do you need to have it `-quiet`? If not lets remove it.

**Review by goncalo-frade-iohk — APPROVED** [review]

**Review by petevielhaber — APPROVED** [review]
